web_app/telegram/handlers/command.py

```diff
@@ -0,0 +1,40 @@
+from aiogram import Router, types
+from aiogram.filters import CommandStart, CommandObject, Command
+from sqlalchemy import update
+from sqlalchemy.orm import Session
+
+from database.models import NotificationData
+from .utils import kb
+
+cmd_router = Router()
+
+
+@cmd_router.message(Command("menu"))
+async def menu(message: types.Message):
+    """
+    This function is triggered when the user sends the "/menu" command to the bot.
+    It sends the user a message "Menu:" along with a reply markup containing a menu with buttons.
+    """
+    await message.answer("Menu:", reply_markup=kb.menu())
+
+
+@cmd_router.message(CommandStart(deep_link=True, deep_link_encoded=True))
+async def start(message: types.Message, db: Session, command: CommandObject):
+    """
+    Register Telegram ID in the database.
+
+    :param message: The Telegram message, typically a "/start" command.
+    :param db: The database session.
+    :param command: The extracted data from the CommandStart filter.
+    """
+    ident = command.args
+    stmp = (
```

Comment:
I think it should be in `crud.py` file. it's more appropriate place for working with db